### DISCUSSION TITLE
Set Real-time date and time

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#3925] [Help]
Hi! I'm new to budibase. I just want to ask if can we set the date and time real-time? like if I have a "date" field in my form, instead of setting the date manually, it will automatically set the date and time real-time. 

Hoping someone will answer me. I need it for a project. Thanks!

**Comment by mjashanks** [comment]
Hi @lykaodan - welcome!

There are 2 ways to go about this
1. Add a Date Picker and set the `Default Date` to `{{ now }}`
![image](https://user-images.githubusercontent.com/3524181/148214738-a1959059-ac6a-4ee8-8e66-b989cd00b8c0.png)

This means that the date will be set, but it can be overridden by the user.

2. Set the date in the background, using the `Save Row` action

In the "Define Actions" for your save button, you should have a `Save Row` action. Now, "Add Column" and set the value to `{{ now }}`. You may then remove the date picker field from your form.

This will set the date upon saving, without giving the user to override it.

![image](https://user-images.githubusercontent.com/3524181/148215044-59fa1a60-2c5a-4cbf-9d93-4db0454a6b66.png)

**Comment by lykaodan** [comment]
Thank you so much! I'll try this and give you a feedback .
